tools/topology2/include/dais/ssp.conf

```diff
@@ -0,0 +1,134 @@
+#
+# Intel SSP DAI
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "ssp.attribute_name"
+#
+# Usage: this component can be used by declaring in the "dais" field of
+# a parent object. i.e.
+#
+# For Capture
+#Object.SSP."0" {
+#	index			0
+#	direction		"capture"
+#	stream_name		"NoCodec-0"
+#	id 			0
+#	format			"s24le"
+#	quirks			"lbm_mode"
+#	sample_bits		24
+#	Object.hw_config."0" {
+#		mclk_freq	24000000
+#		bclk_freq	4800000
+#		tdm_slot_width	25
+#	}
+#	Object.pipeline-dai-capture."0" {
+#		period_source_count	0
+#		period_sink_count	2
+#	}
+#}
+#
+# For Duplex
+#Object.SSP."N" {
+#	index			0
+#	direction		"duplex"
+#	dai_name		"NoCodec-0"
+#	id 			0
+#	format			"s24le"
+#	quirks			"lbm_mode"
+#	sample_bits		24
+#	Object.hw_config."0" {
+#		mclk_freq	24000000
+#		bclk_freq	4800000
+#		tdm_slot_width	25
+#	}
+#	Object.pipeline-dai-playback."0" {
+#		period_source_count	2
+#		period_sink_count	0
+#	}
+#}
+#
+# where N is the unique instance number for the SSP object within the same alsaconf node.
+<include/dais/hw_config.conf>
+
+# SSP port definition
+Class.Dai."SSP" {
```

Comment:
Does this imply any kind of inheritance from `Class.Component."dai"`?

Reply:
the component is for the DAI widget and this is config for the DAI itself